src/working/test_merge_sort.py

```diff
@@ -0,0 +1,43 @@
+"""Testing for inesrt sort."""
+from merge_sort import mergesort
+import pytest
+from random import randint
+
+
+TEST_PARAMS = [
+    ([i for i in range(50)]),
+    ([randint(1, 500) for i in range(50)]),
+    ([5, 9, 1, 2, 6])
+]
+
+
+PARAMS_TABLE = [
```

Comment:
I would like to see you add some more test cases with duplicate numbers and negative numbers as input.